DGN/Models/Branch.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace DGN.Models
+{
+    public class Branch
+    {
+        public int Id { get; set; }
+
+        [Required]
+        [DisplayName("Branch Name")]
+        public string BranchName { get; set; }
+
+        [DisplayName("Phone Number")]
+        public int PhoneNumber { get; set; }
```

Comment:
Can't be `int` since a phone number starts with a `zero` (Or you can use a prefix).
But, phone number validation is a lot of headaches
Instead, use an email address for each branch - for that, we have built-in email validation (Via DataAnnotation)

P.s.
There is also a phone number validation but everyone on the internet keep using regex to validate a phone number so I just prefer an email address